versions.lock

```diff
@@ -358,21 +359,23 @@ org.glassfish.grizzly:grizzly-framework:2.4.4 (3 constraints: bc31e6d7)
 org.glassfish.grizzly:grizzly-http:2.4.4 (1 constraints: e11123e3)
 org.glassfish.grizzly:grizzly-http-client:1.16 (1 constraints: e4172133)
 org.glassfish.grizzly:grizzly-websockets:2.4.4 (1 constraints: 1418374b)
-org.glassfish.jersey.connectors:jersey-grizzly-connector:2.37 (1 constraints: 6d0e8942)
-org.glassfish.jersey.media:jersey-media-jaxb:2.37 (1 constraints: e3195aca)
-org.glassfish.jersey.test-framework:jersey-test-framework-core:2.37 (2 constraints: ba31fa3a)
-org.glassfish.jersey.test-framework.providers:jersey-test-framework-provider-inmemory:2.37 (1 constraints: 6d0e8942)
+org.glassfish.jersey.connectors:jersey-grizzly-connector:2.39.1 (1 constraints: ce0ef65f)
+org.glassfish.jersey.media:jersey-media-jaxb:2.39.1 (1 constraints: 441ab3fe)
+org.glassfish.jersey.test-framework:jersey-test-framework-core:2.39.1 (2 constraints: 7c326de0)
+org.glassfish.jersey.test-framework.providers:jersey-test-framework-provider-inmemory:2.39.1 (1 constraints: ce0ef65f)
 org.gmock:gmock:0.8.3 (1 constraints: 0d050636)
 org.hamcrest:hamcrest-library:2.1 (1 constraints: 1507415c)
 org.jmock:jmock:2.12.0 (2 constraints: 7d0fa3a1)
 org.jmock:jmock-legacy:2.12.0 (1 constraints: 3705353b)
 org.jmock:jmock-testjar:2.12.0 (1 constraints: a507a272)
-org.junit.jupiter:junit-jupiter-api:5.4.2 (2 constraints: 3c219571)
-org.junit.jupiter:junit-jupiter-engine:5.4.2 (1 constraints: 92108bb2)
-org.junit.platform:junit-platform-commons:1.4.2 (2 constraints: d5202f4a)
-org.junit.platform:junit-platform-engine:1.4.2 (2 constraints: ed22c407)
+org.junit.jupiter:junit-jupiter:5.9.1 (2 constraints: 923d8556)
+org.junit.jupiter:junit-jupiter-api:5.9.1 (4 constraints: 0740843c)
+org.junit.jupiter:junit-jupiter-engine:5.9.1 (2 constraints: 9d1e4185)
+org.junit.jupiter:junit-jupiter-params:5.9.1 (1 constraints: 0c0ee13b)
+org.junit.platform:junit-platform-commons:1.9.1 (2 constraints: dd200f4b)
+org.junit.platform:junit-platform-engine:1.9.1 (2 constraints: f1229e08)
 org.junit.platform:junit-platform-launcher:1.4.2 (1 constraints: 8e1077b2)
```

Comment:
feels weird that this dep isn't getting bumped with the rest of the `org.junit.platform:junit-platform-*` deps

Reply:
probably warrants a glob in versions.props?